### PR TITLE
add new exit code to sdk (INSUFFICIENT_PRIVILEGES 109)

### PR DESCRIPTION
**Pull request by bfuzzy1**

### CODE REVIEW

**Comment by makk94** [comment]
Did y'all want to add back 110 too?

**Comment by makk94** [comment]
Also would you add to our docs as well https://docs.preludesecurity.com/docs/understanding-results

**Comment by bfuzzy1** [comment]
> Did y'all want to add back 110 too?

cc'ing @smprelude on this one. I don't think we should i can make the pr to remove it from endpoint if Scott agree's. if not we should discuss re-definition as its defined as being health check related really.

**Comment by smprelude** [comment]
let's not necro 110. health-check style tests aren't in active production and so there won't be any reason to have it save for backwards compatibility with old test assets that utilize it.

**Comment by bfuzzy1** [comment]
> let's not necro 110. health-check style tests aren't in active production and so there won't be any reason to have it save for backwards compatibility with old test assets that utilize it.

After some discussion. We should have it on until we refresh / update the agent potentially so I will add it back.

We do have tests that use it currently.

File: python/sdk/prelude_sdk/models/codes.py [hunk 1]
```diff
@@ -112,11 +114,13 @@ def mapping(cls):
                 ExitCode.UNEXPECTED_ERROR,
                 ExitCode.UNKNOWN_ERROR,
                 ExitCode.UNREPORTED,
+                ExitCode.INCORRECTLY_BLOCKED,
```
Comment (makk94) [inline python/sdk/prelude_sdk/models/codes.py:117]: nit: alphabetize these 2 error state mappings

**Review by makk94 — APPROVED** [review]

**Pull request merged**